crates/superposition/organisation/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "organisation"
+version = "0.1.0"
+edition = "2021"
+
+
+[dependencies] 
+service_utils = { path = "../../service_utils" } 
+superposition_types = { path = "../../superposition_types", features = [
+    "result",
+    "diesel_derives",
+]}
+serde = { workspace = true }
+serde_json = { workspace = true }
+chrono = { workspace = true } 
+actix-web = { workspace = true } 
+anyhow = { workspace = true } 
+diesel = { workspace = true , features = ["numeric"]}
+idgenerator = "2.0.0"
```

Comment:
idgenerator seems to use snowflake internally, can we just use the snowflake generator internally?
https://crates.io/crates/idgenerator